_includes/results/round-one-result.html

```diff
@@ -1,57 +1,157 @@
+            <li>{{ role }}</li>
+          {% endfor %}
+          </ul>
+        {% else %}
+          <a href="{{pilot.joa.url}}">Job announcement</a> posted from {{ pilot.joa.open-date }} to {{ pilot.joa.close-date }}.
+        {% endif %}
+        <div class="smeqa-results-card__stat-container">
+          <span class="smeqa-results-card__stat">{{ pilot.applicants.total }}</span> 
+          applied
+        </div>
+      </div>
+    </div>
+  </li>
+  <li class="usa-card usa-card--flag smeqa-results-card">
+    <div class="usa-card__container smeqa-results-card__container">
+      <header class="usa-card__header">
+        <h2 class="usa-card__heading">3. Resume Review</h2>
+      </header>
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img src="{{ site.baseurl }}/assets/img/icon-resume-review.png"
+            alt="Icon showing two individuals reviewing resumes">
+        </div>
+      </div>
+      <div class="usa-card__body">
+        {% assign assessment = pilot.assessments[0] %}
+        <div class="smeqa-results-card__stat-container">
+          <span class="smeqa-results-card__stat">{{ assessment.passed }} ({{ assessment.percentage }}%)</span>
+          passed resume review
+        </div>
+      </div>
+    </div>
+  </li>
+  <li class="usa-card usa-card--flag smeqa-results-card">
+    <div class="usa-card__container smeqa-results-card__container">
+      <header class="usa-card__header">
+        <h2 class="usa-card__heading">4. Assessments</h2>
+      </header>
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img src="{{ site.baseurl }}/assets/img/icon-assessments.png"
+            alt="Icon showing individuals being assessed">
+        </div>
+      </div>
+      <div class="usa-card__body">
+        <ul class="smeqa-results-cards-list">
+        {% assign round-one = pilot.assessments[1] %}
+        {% assign round-two = pilot.assessments[2] %}
+          <li class="smeqa-results-cards-list__item">
+            <div class="smeqa-results-card__stat-container">
+              <span class="smeqa-results-card__stat">
+                {{ round-one.passed }} ({{ round-one.percentage }}%)
+              </span>
+              passed {{ round-one.name | downcase }}
+            </div>
+          </li>
+          <li class="smeqa-results-cards-list__item">
+            <div class="smeqa-results-card__stat-container">
+              <span class="smeqa-results-card__stat">
+                {{ round-two.passed }} ({{ round-two.percentage }}%)
+              </span>
+              passed {{ round-two.name | downcase }}
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </li>
+  <li class="usa-card usa-card--flag smeqa-results-card">
+    <div class="usa-card__container smeqa-results-card__container">
+      <header class="usa-card__header">
+        <h2 class="usa-card__heading">5. Issuing a Certificate</h2>
```

Comment:
```suggestion
        <h2 class="usa-card__heading">5. Selections from Certificate(s)</h2>
```
Because the metrics at this stage are about selections, I think we should change the title to be clearer - the impression I get from this this is that the cert was issued with the number, not that the number reflects selections from a larger cert.